src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -156,26 +122,14 @@
                     return surveyAndProgram.selectedDataType != null;
                 },
                 anc: ({surveyAndProgram}: {surveyAndProgram: SurveyAndProgramState}) => ({
-                    valid: !!surveyAndProgram.anc,
-                    fromADR: !!surveyAndProgram.anc?.fromADR,
-                    error: surveyAndProgram.ancError,
-                    existingFileName: (surveyAndProgram.anc && surveyAndProgram.anc.filename)|| surveyAndProgram.ancErroredFile,
                     available: !surveyAndProgram.ancError && surveyAndProgram.anc
-                } as PartialFileUploadProps),
+                } as any),
```

Comment:
Not sure you even need the `as any` here - given there's no need to explicitly broaden the type

Reply:
You are right, any removed.